docs/FAQ.md

```diff
@@ -5,3 +5,37 @@ This FAQ is in progress. If there is anything you'd like to see added, please re
 ##### What is the difference between cron, batchtime, and periodic build?
 
 The main difference is that cron and batchtime are tied to activating builds for existing commits and periodic builds creates new builds on a schedule regardless of commit activity. For more on their differences and examples, see [controlling when tasks run](Project-Configuration/Controlling-when-tasks-run).
+
+#### Dynamic Github Access Tokens
+
+#### Can more than one app be associated with my project?
```

Comment:
I think since this file is so small, we should redo the #'s. Like the above question that exists, being three ## and then the title Dynamic GitHub access tokens being 3 as well, then these being 4. I'm fine with not exactly that, but I don't think they should be the same as the title (both at 4 # right now)